app/models/event.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class Event < ApplicationRecord
+  # default_scope { order(date: :asc) }
+  scope :past_events, lambda {
```

Comment:
Consider using `Time.zone.now` that would give a more accurate reading I think 🤔 
[OPTIONAL]